docker/docker-compose.yml

```diff
@@ -0,0 +1,178 @@
+version: "3.8"
+name: hskauting
+
+networks:
+    traefik:
+
+volumes:
+    mysql:
+    mysql-test:
+
+x-php-base:
```

Comment:
co znamená prefix `x-`?

Reply:
x- je mnou zvolená předpona pro předchůdce.

Reply:
to jsem chápal, ze je vyjádřeno tím `base`. Možná tedy upravit na prefix `base-`? samotné X me nenapadlo co by melo znamenat

Reply:
Jo už vím proč je tam x- :-) 
ono je to z dokumentace. https://nickjanetakis.com/blog/docker-tip-82-using-yaml-anchors-and-x-properties-in-docker-compose#:~:text=The%20same%20example%20using%20YAML%20Anchors%20and%20the%20X%20prefix%20to%20set%20defaults%3A